tezos.go

```diff
@@ -0,0 +1,50 @@
+package coincodec
+
+import (
+	"errors"
+	"bytes"
+	"github.com/wealdtech/go-slip44"
+)
+
+const (
+	AddressLength = 23
```

Comment:
this would expose a general `AddressLength` outside this file, let's add `tezos` prefix to avoid that

Reply:
done